diff_drive_controller/CMakeLists.txt

```diff
@@ -3,13 +3,14 @@ project(diff_drive_controller)
 
 find_package(catkin REQUIRED COMPONENTS
     controller_interface
-    urdf
+    nav_msgs
     realtime_tools
     tf
-    nav_msgs)
+    urdf
+)
 
 catkin_package(
-  INCLUDE_DIRS include
```

Comment:
Thanks. I really want to push all the fork changes upstream... I just need some time and a plan of how to do it, since there are many different new features.